tools/devtool

```diff
@@ -534,6 +534,7 @@ cmd_test() {
         --security-opt seccomp=unconfined \
         --ulimit core=0 \
         --workdir "$CTR_FC_ROOT_DIR/tests" \
+        -e KEEP_TEST_SESSION="$KEEP_TEST_SESSION" \
```

Comment:
Just propagating the env variable here is not enough.

The `test_session` dir is configured somewhere in `/tmp`. That dir is perishable and will go away along with the `docker session` (`tools/devtool test` runs the container and ends its session once tests complete).

In the current PR form, this line is not useful.

I suggest you either:
1. Extend `devtool` to be able to actually keep the test session. This implies mounting the `test-session-dir` in the container from somewhere on the host (preferably `/tmp/smth`) the same way the `firecracker` dir is mounted to get the source files.
2. Remove this line to merge this PR to enable support for keeping the test session when running without the container, and open an issue to enhance `devtool`.

Note: One can use the `KEEP_TEST_SESSION` functionality even for testing in the container like so:
```bash
firecracker$ tools/devtool shell -p
[Firecracker devtool] Dropping to a privileged shell prompt ...
[Firecracker devtool] Note: /home/ANT.AMAZON.COM/acatan/code/firecracker is bind-mounted under /firecracker
[Firecracker devtool] You are running as root; any files that get created under /firecracker will be owned by root.
root@4a8b30e393fc:/firecracker# cd tests/
root@4a8b30e393fc:/firecracker/tests# KEEP_TEST_SESSION=1 ; pytest
/// running tests but not deleting the test session at the end

root@4a8b30e393fc:/firecracker/tests# pytest
/// running tests again, only doing incremental rebuild of binaries if any src files changed
```

Reply:
Thank you for raising this, @acatangiu. I did not think about that. 
Hopefully it's now better.